src/main/python/analyzers/project_analyzer.py

```diff
@@ -20,26 +39,32 @@ def getSentimentAnalysis(self, tweetsList: list.__class__, regression=True):
         # if regression:
         #     resultsDictOrDF = __performRegressionAnalysis(resultsDF, featureColumnIndexes))
         # else:
-        #     resultsDictOrDF = __performOtherAnalysis(resultsDF, featureColumnIndexes)
+        #     resultsDictOrDF = self.__performRandomForestAnalysis(resultsDF, featureColumnIndexes)
         # return resultsDictOrDF
 
-    def getEmotionAnalysis(self, regression=True):
+    def getEmotionAnalysis(self, regression='1'):
```

Comment:
i thought so as well, but then again, all the models we are using are in fact doing regression (rather than classification), which is why i decided that it is not such a bad thing after all. i know the word 'regression' is generally interpreted as linear regression, but technically speaking, any model that predicts a number instead of a class is a regression model. to avoid confusion, i'll change it to regressionMethod

Reply:
Interesting, I did not know that. Thanks!

Reply:
just pushed the updated code